### PR TITLE
Add i18n and russian translation

### PR DESCRIPTION
**Pull request by diastremskii**
Atom changed last lines a bit, sorry about that.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "name": "chip-in-calculator",
   "version": "1.1.1",
-  "authors": ["Alexander Kasheverov"],
+  "authors": [
+    "Alexander Kasheverov"
```
Comment (kashesandr) [inline bower.json:5]: @TheBeastOfCaerbannog don't you want to add yourself here? :)

File: src/index.html [hunk 2]
```diff
@@ -36,8 +36,21 @@
         }
     </script>
     <!-- /Google Analytics counter -->
+    <!-- i18n -->
+    <script>
+        document.addEventListener('HTMLImportsLoaded', function() {
```
Comment (kashesandr) [inline src/index.html]: Can we move this part inside a related Polymer-element?

Comment (diastremskii) [inline src/index.html]: Sure, something new like cic-i18n or in cic-main?
(Well, something like commit 69037d69a2f246a5cfb9797974a5551366790f43?)

File: bower.json (hunk 1 above)
Comment (diastremskii) [inline bower.json:5]: Nah, I simply added translation :)

**Comment by kashesandr** [comment]
cc @bearded-maks

**Pull request merged**